src/Sulu/Bundle/RouteBundle/Tests/Unit/DependencyInjection/RouteGeneratorCompilerPassTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\RouteBundle\Tests\Unit\DependencyInjection;
+
+use Prophecy\Argument;
+use Sulu\Bundle\RouteBundle\DependencyInjection\RouteGeneratorCompilerPass;
+use Sulu\Bundle\RouteBundle\Generator\RouteGeneratorInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class RouteGeneratorCompilerPassTest extends \PHPUnit_Framework_TestCase
```

Comment:
Possible that these tests have also to be adapted when we use aliases instead of services?